src/test/java/com/pgs/booking/controller/PassengerControllerTest.java

```diff
@@ -82,6 +100,37 @@ void testGetSinglePassenger() throws Exception {
         verify(passengerService).getSinglePassenger(5L);
     }
 
+    @WithMockUser
+    @Test
+    void testAddPassenger() throws Exception {
+        when(passengerService.addPassenger(any(CreateUpdatePassengerDto.class))).thenReturn(CREATE_UPDATE_PASSENGER_DTO);
+        mockMvc.perform(post("/api/passengers")
+                        .content(objectMapper.writeValueAsString(CREATE_UPDATE_PASSENGER_DTO))
+                        .contentType(MediaType.APPLICATION_JSON)
+                        .accept(MediaType.APPLICATION_JSON))
+                .andExpect(status().isOk());
+              //.andExpect(jsonPath("$.id").exists());
```

Comment:
Proszę o wskazówkę dlaczego to nie działa.
Jeżeli zmienię to na inne pole firstName lub dowolne z pozostałych jest ok, ale wygląda tak, jakbym do id nie miał dostępu albo nie istniało w jsonie.